app/src/main/res/values-zh-rCN/arrays.xml

```diff
@@ -3,13 +3,13 @@
     <string-array name="background_play_entries">
         <item>从不</item>
         <item>仅音频</item>
-        <item>总是(视频文件同)</item>
+        <item>总是(及视频)</item>
     </string-array>
     <string-array name="auto_rotation_entries">
         <item>自动</item>
-        <item>默认横屏</item>
-        <item>默认竖屏</item>
-        <item>Device</item> <!-- TODO -->
+        <item>默认横向</item>
+        <item>默认纵向</item>
+        <item>设备</item> <!-- TODO -->
```

Comment:
`<!-- TODO -->` was supposed to indicate that this translation is incomplete. You can remove it.